public/components/map_top_nav/top_nav_menu.tsx

```diff
@@ -108,7 +127,7 @@ export const MapTopNavMenu = ({
 
   return (
     <TopNavMenu
-      appName={PLUGIN_NAVIGATION_BAR_ID}
+      appName={MAPS_APP_ID}
       config={getTopNavConfig(services, {
```

Comment:
nit: Should we memo the config so we don't need to recompute the returned value of getTopNavConfig each time when we re-render unless it is changed? I think `getTopNavConfig` is a pretty long and complex computation...might worth to memorize it.

```
const config = useMemo(() => { 
 return getTopNavConfig(services, {
        mapIdFromUrl,
        layers,
        title,
        description,
        setTitle,
        setDescription,
        mapState,
        originatingApp
});
}, [mapIdFromUrl,layers,title,description,setTitle,setDescription,mapState,originatingApp]);
```
Then 

```
<TopNavMenu
      appName={MAPS_APP_ID}
      config={config}
      ...
/>

```

Reply:
Yes, updated to use memo.